src/wp-includes/script-loader.php

```diff
@@ -2730,6 +2735,44 @@ static function( $a, $b ) {
 	}
 }
 
+/**
+ * Make URLs relative to the WordPress installation.
+ *
+ * @since 5.8.2
+ *
```

Comment:
```suggestion
 * @access private
```

To denote this global function is intended for use within core itself. Though it's a publicly available for use outside of core, marking it as `private` means there's not a BC guarantee into the future.